src/main/java/core/basesyntax/OperationHandler.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+import java.util.Map;
+
+public interface OperationHandler {
+        void handle(FruitTransaction transaction, Map<String, Integer> storage);
+
+}
```

Comment:
The `OperationHandler` interface should be placed in the `strategy` package according to the checklist item 'Don't keep all logic in a single package'. This helps in organizing the code better and making it easier to navigate.